README.md

```diff
@@ -14,8 +14,8 @@ In order to understand the library, one must first be familiar with the
 
 OpenTracing-PHP can be installed via Composer:
 
-```php
-composer require jcchavezs/opentracing-php
+```bash
+composer require opentracing/opentracing-php
```

Comment:
the `-php` seems redundant here, all package on Composer are in PHP.
`opentracing-javascript` is just `opentracing` on npm for the same reason.